### PR TITLE
Set up MongoDB in Review micro-service

### LINKED ISSUE #26: Set up MongoDB in Review micro-service
**Description**
MongoDB usage in Review micro-service

**Why is this needed**
This is needed so we can store reviews in MongoDB

**Tasks**
- [x] ~~Make Custom `ReviewId` (maybe)~~
- [x] Add annotations to `Review`
- [x] Add `ReviewRepository` in correct folder

### PR DESCRIPTION
**Pull request by xandervedder**
## Changes
Added MongoDb annotations to Review domain objects.

## Checklist
- [x] Add MongoDb annotations
- [x] Tested by making a TestController (not included in PR)

## When merging, this pull request 
closes #26

### CODE REVIEW

File: review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java [hunk 1]
```diff
@@ -13,7 +15,8 @@
     private static final int MIN_DESCRIPTION_LENGTH = 32;
     private static final int MAX_DESCRIPTION_LENGTH = 512; // Is this not enough? Should it be more?
```
Comment (JustMilan) [inline review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java:16]: Is this question still relevant?

**Review by JustMilan — APPROVED** [review]

File: review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java (hunk 1 above)
Comment (xandervedder) [reply review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java:16]: It isn't, should be removed :+1:

File: review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java [hunk 2]
```diff
@@ -11,9 +13,10 @@
     private static final int MIN_TITLE_LENGTH = 3; // Allows for titles like 'Bad' and 'Good'
```
Comment (JortWillemsen) [inline review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java:13]: Is this really what you want? I don't think bad or good are valid review titles and should not be allowed.

**Review by JortWillemsen — APPROVED** [review]

File: review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java (hunk 2 above)
Comment (xandervedder) [reply review/src/main/java/nl/softwarestrijders/waiter/review/core/domain/ReviewBase.java:13]: Could take a look at this in a different PR.

**Pull request merged**